magnum_capi_helm/driver.py

```diff
@@ -550,6 +561,8 @@ def _update_helm_release(self, context, cluster, nodegroups=None):
 
         network_id = self._get_fixed_network_id(context, cluster)
         subnet_id = neutron.get_fixed_subnet_id(context, cluster.fixed_subnet)
+        kubenetwork_pod_cidr = "172.16.0.0/13"
```

Comment:
Should this come from config, as for some people this might conflict with other internal services? I liked delegating this to the helm chart I guess.

Reply:
Isn't there a label that these should come from?

Reply:
Ack, I had left this in as I figured it was a no-op. But it isn't - the helm chart can't be modified with this change. I'll remove it from this PR.

There is an existing `calico_ipv4pool` label, and maybe others.

I'll move this to a separate PR and pull in some labels for these instead of hardcoding.

Reply:
@dalees ah, good idea, lets split this one out. Thank you!

Reply:
Moved `kubenetwork` changes to https://github.com/stackhpc/magnum-capi-helm/pull/30